include/lora.h

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
We are trying to migrate away from single include files in include/ directory. The lora.h could be moved to include/net as there is also other radio technology headers there.

Reply:
Okay

Reply:
@jukkar IMO this belongs to include/drivers.

Reply:
> @jukkar IMO this belongs to include/drivers.

Could be, we already have IEEE 802.15.4 generic (non-driver) related headers in include/net, so in this respect the lora.h could be placed there. If the lora.h contains very driver specific stuff, then sure, include/driver is better place.

Reply:
@jukkar for me it looks like a driver ;-)

Reply:
Thinking about how this API will be used in future, I'd like it to be in `include/net`. Currently, it just deals with only plain driver stuff but for sure it will change soon.

Reply:
Ok. Thought this API is finished and the networking related stuff goes into a separate API.

Reply:
> Thought this API is finished and the networking related stuff goes into a separate API.

TBH there is no clear separation of drivers/lora now. That's why I thought of placing it under `net`. Till we have the LoRaWAN implementation it would be difficult to guess how this API will evolve.